Benchmarks/BuildingSimulation/Haskell/building-simulation.cabal

```diff
@@ -0,0 +1,31 @@
+cabal-version:      3.0
+name:               building-simulation
+version:            0.1.0.0
+-- synopsis:
+-- description:
+-- license:            BSD-3-Clause
+-- license-file:       LICENSE
+author:             Trevor L. McDonell
+maintainer:         trevor@passivelogic.com
+-- copyright:
+build-type:         Simple
+-- extra-doc-files:    README.md CHANGELOG.md
+-- extra-source-files:
+
+executable building-simulation
+    default-language: Haskell2010
+    hs-source-dirs:   app
+
+    main-is:          Main.hs
+    -- other-modules:
+
+    ghc-options:
+      -O2
+      -Wall
+
+    build-depends:
+        base          ^>= 4.20
```

Comment:
I was able to run this successfully after dropping to `4.17`. Unless we need something from `4.20` specifically, maybe we lower the requirement to the recommended version.

Reply:
Oh yeah it seems we aren't using anything from 4.20; this would have just been the default it filled in when I typed in `cabal init` to create a new project skeleton. I'll drop it.